cmd/humioctl/ingest.go

```diff
@@ -36,7 +36,7 @@ func tailFile(filepath string, quiet bool, seekToEnd bool, handler shipper.LineH
 	for line := range t.Lines {
 		handler.HandleLine(line.Text)
 		if !quiet {
-			fmt.Println(line.Text)
+			log.Println(line.Text)
```

Comment:
This is output from the command.

```suggestion
			fmt.Fprintln(cmd.OutOrStdOut(), line.Text)
```